app/renderer/js/pages/preference/shortcuts-section.js

```diff
@@ -130,6 +130,14 @@ class ShortcutsSection extends BaseSection {
                     <td><kbd>Option</kbd><kbd>${userOSKey}</kbd><kbd>U</kbd></td>
                     <td>Toggle DevTools for Active Tab</td>
                   </tr>
+                  <tr>
+                    <td><kbd>${userOSKey}</kbd> + <kbd>Tab</kbd></td>
+                    <td>Switch to next organization</td>
+                  </tr>
+                  <tr>
+                    <td><kbd>${userOSKey}</kbd> + <kbd>Shift</kbd> + <kbd>Tab</kbd></td>
```

Comment:
Same.

Reply:
I've pushed the changes needed. I don't have access to a Mac right now, so I can't check for myself if the section looks as expected. Please let me know if it doesn't.